app/general/models.py

```diff
@@ -84,36 +101,49 @@ class DocumentFile(models.Model):
                     More information about Creative Commons licenses.
             </a>'
     """
-    document_type_choices = [("Glossary", "Glossary"), ("Policy", "Policy")]
+    document_type_choices = [("Glossary", _("Glossary")), ("Policy", _("Policy"))]
 
     file_type = "pdf"
 
-    title = models.CharField(max_length=200)
-    url = models.URLField(max_length=200, blank=True, verbose_name="URL")
+    title = models.CharField(max_length=200, verbose_name=_("title"))
+    url = models.URLField(max_length=200, blank=True, verbose_name=_("URL"))
     uploaded_file = models.FileField(
         upload_to="documents/",
         validators=file_validators,
         blank=True,
-        help_text="PDF files up to 10MB are allowed.",
+        help_text=_("PDF files up to 10MB are allowed."),
+        verbose_name=_("uploaded file"),
     )
-    available = models.BooleanField(default=True)
+    available = models.BooleanField(default=True, verbose_name=_("available"))
     license = models.CharField(
         max_length=200,
         choices=license_choices,
         default="(c)",
         help_text=license_help_text,
+        verbose_name=_("license"),
     )
     mime_type = models.CharField(
-        max_length=200, blank=True, help_text="This input will auto-populate."
+        max_length=200,
+        blank=True,
+        help_text=_("This input will auto-populate."),
+        verbose_name=_("mime type"),
```

Comment:
While we're busy, let's write this one correct as "MIME type". I realise it didn't have this verbose name before.